config/test.js

```diff
@@ -49,6 +49,23 @@ function makeTasks(
     return ["env", "SOURCECRED_BIN=" + backendOutput, ...invocation];
   }
 
+  function flowCommand(limitMemoryUsage /*: boolean */) {
+    const cmd = [
+      "yarn",
+      "run",
+      "--silent",
+      "flow",
+      "--quiet",
+      "--max-warnings=0",
+    ];
+    // Use only one worker to try to avoid flow flakey failures
+    if (limitMemoryUsage) {
+      cmd.push("--flowconfig-name");
+      cmd.push(".flowconfig-ci");
```

Comment:
nit: This can be `cmd.push("--flowconfig-name", ".flowconfig-ci")`.

Reply:
But if you prefer it as is, no objection hence.

Reply:
Thanks, I thought there was probably a slightly cleaner way to do this. Change accepted.